requirements/pip.in

```diff
@@ -79,19 +79,14 @@ orjson
 django-gravatar2
 pytz
 
-# We cannot upgrade it until dj-stripe fixes this issue
-# https://github.com/dj-stripe/dj-stripe/issues/1842
-stripe==4.2.0
-
 regex
 
-# NOTE: djstripe overrides the stripe api version
```

Comment:
We should keep this note, but we use the stripe package directly in .org.

Reply:
The newer version says this is not like that anymore. There is a note in the release notes. 

Maybe we should define it explicitly now?

Reply:
I didn't find any mention of that, can you link me to that?

Reply:
Yes. I'll share it once I get to my laptop

Reply:
It's under "Other changes" at https://github.com/dj-stripe/dj-stripe/releases/tag/2.8.0

> stripe.api_version is no longer manipulated by dj-stripe.